app/services/check_answers/section_id_finder.rb

```diff
@@ -3,6 +3,7 @@ class SectionIdFinder
     IDS = {
       level_of_help: "section-level-of-help-header",
       tribunal: "subsection-case_details-header",
+      asylum_support: "subsection-case_details-header",
```

Comment:
I think you want this to be "subsection-asylum_support-header" - that way when you return to this section after clicking the asylum support change link, you're brought to exactly the right subsection you clicked on.

Reply:
updated, weirdly both work and take you to the correct page 🤷